bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java

```diff
@@ -141,6 +141,10 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
 
         reply.println(ReviewersTracker.setReviewersMarker(numReviewers, role));
         var totalRequired = formatLimits.values().stream().mapToInt(Integer::intValue).sum();
+        if (pr.labelNames().contains("clean") && pr.labelNames().contains("backport")) {
+            reply.println("Warning: The /reviewers command is used in a clean backport, it will enable reviewers check for this pr. " +
+                    "If the reviewers check requirements in the jcheck conf are stricter than your requirements, the bot will enforce the stricter requirements.");
```

Comment:
```suggestion
            reply.println("Warning: By issuing the /reviewers command in this clean backport pull request, the reviewers check has now been enabled.");
```
I don't think we need to point out the application of stricter requirements in the reply. It's enough to document this on the wiki. I see I asked for it in the bug, but thinking on this more, the /reviewers command has never been able to reduce reviewer requirements from the minimum specified in .jcheck/conf.

Reply:
> I don't think we need to point out the application of stricter requirements in the reply. It's enough to document this on the wiki. I see I asked for it in the bug, but thinking on this more, the /reviewers command has never been able to reduce reviewer requirements from the minimum specified in .jcheck/conf.

Yes, it's better. Thx!